index.js

```diff
@@ -141,6 +144,12 @@ File.prototype.inspect = function() {
   return '<File ' + inspect.join(' ') + '>';
 };
 
+File.builtInFields = ['_contents', 'contents', 'stat', 'history', 'path', 'base', 'cwd'];
```

Comment:
I'll just chime in and say that exposing it as a static property on the `File` constructor is my preferred solution. It is a clean way for supporting other solutions that extend `Vinyl` as is demonstrated here. If it was added to the `prototype`, it would have to be stepped around during clone and such to prevent clobbering. However, adding it as an argument to the constructor itself would require a sub-class to specify those additional args on each instance. Depending on how it's done, that becomes an extra burden to either the devs or the consumers. (not to mention having tons of duplicate arrays floating around in memory)